framework-examples/react/src/Checkbox.js

```diff
@@ -101,6 +110,45 @@ export default class Checkbox extends PureComponent {
     isAttachedToDOM: () => Boolean(this.refs.nativeCb),
   });
 
+  // For browser compatibility we extend the default adapter which checks for css variable support.
+  rippleFoundation = new MDCRippleFoundation(Object.assign(MDCRipple.createAdapter(this), {
+    isUnbounded: () => true,
+    isSurfaceActive: () => this.refs.root[MATCHES](':active'),
```

Comment:
Makes sense! I was wondering why it's the root. I was using the native js version that you fixed as my guide. Thanks for clearing that up.